components/disabled/index.js

```diff
@@ -83,11 +85,15 @@ class Disabled extends Component {
 
 	render() {
 		return (
-			<div ref={ this.bindNode } className="components-disabled">
-				{ this.props.children }
-			</div>
+			<Provider value={ true }>
```

Comment:
In my testing, if we set the default value for context above to `true`, there's no need to pass `value` at all.

i.e.

```js
const { Consumer, Provider } = createContext( true );

// ...

return <Provider>{ /* ... */ }</Provider>;
```

Demo: https://codepen.io/aduth/pen/KeNGjL